changelog.d/2724.fixed.md

```diff
@@ -0,0 +1 @@
+Make `KubeResourceSeeker` to list all resources one at a time and not in parallel, used by `mirrord ls`.
```

Comment:
Make it more user friendly:
`Fix `mirrord ls` hanging caused by `KubeReosurceSeeker` listing all resources in parallel by making it sequential`

Reply:
Updated